content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md

```diff
@@ -1,3 +1,4 @@
+9
```

Comment:
aleksandrpidgurskij:patch-2

Reply:
content/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account.md